frontend/src/views/settings/Profile.vue

```diff
@@ -89,6 +114,7 @@ export default {
       singleClick: false,
       dateFormat: false,
       locale: "",
+      editorAutoSaveInterval: 60,
```

Comment:
Why 60 is being used as the default value? What's the default behavior for existing users?